src/main/java/org/dita/dost/store/Store.java

```diff
@@ -62,7 +97,7 @@
     Source getSource(URI path);
 
     /**
-     * Get immutable DOM document for file. If mutating methods are called,
+     * Get immutable DOM document for resouce. If mutating methods are called,
```

Comment:
Really trivial comment, would be good to fix the typo "resouce", here and a few other spots

Reply:
Fixed